src/clojure_turtle/core.cljc

```diff
@@ -231,28 +230,39 @@
 (defn draw
   "The function passed to Quil for doing rendering."
   []
-  (q/with-translation [(/ (q/width) 2) (/ (q/height) 2)]
-    (reset-rendering)
-    
-    (q/push-matrix)
-    (q/apply-matrix 1  0 0
-                    0 -1 0)
-    (doseq [l @lines]
-      (let [[[x1 y1] [x2 y2]] l]
-        (q/line x1 y1 x2 y2))) 
-    (draw-turtle) 
-    (q/pop-matrix)))
+  (q/push-matrix)
+  (q/translate (/ (q/width) 2) (/ (q/height) 2))
+  (reset-rendering)
+  (q/push-matrix)
+  (q/scale 1.0 -1.0)
```

Comment:
the apply-matrix call in the current code will adjust the coordinates so that we use a math-style grid (positive x axis goes right, positive y axis goes up).  is that what the scale call achieves?

out of curiosity, this diff hunk of new code seems to be functionally equivalent to the old -- is the new code there to support CLJS in a way that the old doesn't?